charts/trino/templates/configmap-catalog.yaml

```diff
@@ -7,13 +8,8 @@ metadata:
     {{- include "trino.labels" . | nindent 4 }}
     app.kubernetes.io/component: catalogs
 data:
-  tpch.properties: |
-    connector.name=tpch
-    tpch.splits-per-node=4
-  tpcds.properties: |
-    connector.name=tpcds
-    tpcds.splits-per-node=4
-{{- range $catalogName, $catalogProperties := .Values.additionalCatalogs }}
```

Comment:
Removing `additionalCatalogs` is a breaking change. Can we avoid this and instead deprecate it first? We can stop documenting it in `values.yaml`, but check if it was set.

Reply:
I added a deprecation notice in the documentation. Let me know if you like it, otherwise I can remove it completely.

Reply:
It's OK, ale just remove the examples (it's the same as in catalogs) and put catalogs first.

Reply:
Done!